internal/forms.py

```diff
@@ -36,24 +36,25 @@ class Meta:
             'committees': SemanticMultipleSelectInput(prompt_text=_("Choose committees")),
         }
 
-    def __init__(self, user, **kwargs):
-        super().__init__(**kwargs)
+    def clean_card_number(self):
```

Comment:
This is probably corrent, I'm just confused since i cant find a function call for this function in the codebase

Reply:
Good question - this is part of some Django "magic" for cleaning specific form fields 😅 See this excerpt from [the docs](https://docs.djangoproject.com/en/3.2/ref/forms/validation/):
> The `clean_<fieldname>()` method is called on a form subclass – where `<fieldname>` is replaced with the name of the form field attribute. This method does any cleaning that is specific to that particular attribute, unrelated to the type of field that it is. [...]

Reply:
Ahh, amazing! Thank you!